index.html

```diff
@@ -88,7 +88,8 @@ <h1>B&auml;rner Go Meetup</h1>
     <div>
       <span>Many thanks you to our...</span>
       <h2 style="color: gold; margin-block-end: 0em; margin-block-start: 0em;">Gold Sponsor</h2>
```

Comment:
As @ghouscht has observed, we do not have a gold sponsor tier. Should we fix this as well?

Reply:
Yes we can fix this as well. Personally I would drop the sponsoring tier here as anyway there is only one which includes the website logo.

So "Many thanks to our ... Sponsors"?

Reply:
Fine for me.